src/components/bcros/businessDetails/Status.vue

```diff
@@ -1,28 +1,20 @@
 <script setup lang="ts">
 import { CorpTypeCd, FilingTypes } from '@bcrs-shared-components/enums'
-import { BusinessStateE } from '~/enums/business-state-e'
-import { useBcrosBusiness } from '~/stores/business'
-import { FilingSubTypeE } from '~/enums/filing-sub-type-e'
-
-// todo: when temp is done
-// const tempRegNumber = (): string => {
-//   return sessionStorage.getItem('TEMP_REG_NUMBER')
-// }
 
 const t = useNuxtApp().$i18n.t
 const {
   currentBusiness,
+  currentBusinessIdentifier,
   stateFiling,
   isInLimitedRestoration,
   isAuthorizedToContinueOut
 } = storeToRefs(useBcrosBusiness())
 
 const getReasonText = computed(() => {
-  const enDash = '–' // ALT + 0150
   if (currentBusiness.value.state !== BusinessStateE.HISTORICAL) {
     return ''
   }
-
+  const enDash = '–' // ALT + 0150
```

Comment:
Is there a reason this has to be an endash instead of a regular dash

Reply:
This is pulled from the old cold base I think so it must've been a design choice at some point to use the bigger dash